CodePush.m

```diff
@@ -177,6 +179,18 @@ - (void)startRollbackTimer:(int)rollbackTimeout
     resolve(@(isFailedHash));
 }
 
+RCT_EXPORT_METHOD(isFirstRun:(NSString *)packageHash
+                  resolve:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    NSError *error;
+    BOOL isFirstRun = (nil != packageHash
+                  && [packageHash length] > 0
+                  && [packageHash isEqualToString:[CodePushPackage getCurrentPackageHash:&error]]
```

Comment:
It looks like getCurrentPackageHash and getPreviousPackageHash would both null dereference if there is an error.